[
  {
    "url": "https://api.github.com/repos/pandas-dev/pandas/issues/comments/21296772",
    "html_url": "https://github.com/pandas-dev/pandas/pull/4305#issuecomment-21296772",
    "issue_url": "https://api.github.com/repos/pandas-dev/pandas/issues/4305",
    "id": 21296772,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxMjk2Nzcy",
    "user": {
      "login": "cpcloud",
      "id": 417981,
      "node_id": "MDQ6VXNlcjQxNzk4MQ==",
      "avatar_url": "https://avatars1.githubusercontent.com/u/417981?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/cpcloud",
      "html_url": "https://github.com/cpcloud",
      "followers_url": "https://api.github.com/users/cpcloud/followers",
      "following_url": "https://api.github.com/users/cpcloud/following{/other_user}",
      "gists_url": "https://api.github.com/users/cpcloud/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/cpcloud/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/cpcloud/subscriptions",
      "organizations_url": "https://api.github.com/users/cpcloud/orgs",
      "repos_url": "https://api.github.com/users/cpcloud/repos",
      "events_url": "https://api.github.com/users/cpcloud/events{/privacy}",
      "received_events_url": "https://api.github.com/users/cpcloud/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-07-20T17:05:29Z",
    "updated_at": "2013-07-20T17:05:29Z",
    "author_association": "MEMBER",
    "body": "cc @jtratner\n#4303 \n"
  },
  {
    "url": "https://api.github.com/repos/pandas-dev/pandas/issues/comments/21302354",
    "html_url": "https://github.com/pandas-dev/pandas/pull/4305#issuecomment-21302354",
    "issue_url": "https://api.github.com/repos/pandas-dev/pandas/issues/4305",
    "id": 21302354,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxMzAyMzU0",
    "user": {
      "login": "jtratner",
      "id": 1598379,
      "node_id": "MDQ6VXNlcjE1OTgzNzk=",
      "avatar_url": "https://avatars1.githubusercontent.com/u/1598379?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/jtratner",
      "html_url": "https://github.com/jtratner",
      "followers_url": "https://api.github.com/users/jtratner/followers",
      "following_url": "https://api.github.com/users/jtratner/following{/other_user}",
      "gists_url": "https://api.github.com/users/jtratner/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/jtratner/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/jtratner/subscriptions",
      "organizations_url": "https://api.github.com/users/jtratner/orgs",
      "repos_url": "https://api.github.com/users/jtratner/repos",
      "events_url": "https://api.github.com/users/jtratner/events{/privacy}",
      "received_events_url": "https://api.github.com/users/jtratner/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-07-20T23:34:06Z",
    "updated_at": "2013-07-20T23:34:06Z",
    "author_association": "CONTRIBUTOR",
    "body": "@cpcloud handling network IO is clearly tricky. Were you able to recreate the conditions for the error that was initially raised with the original URL?\n"
  },
  {
    "url": "https://api.github.com/repos/pandas-dev/pandas/issues/comments/21302395",
    "html_url": "https://github.com/pandas-dev/pandas/pull/4305#issuecomment-21302395",
    "issue_url": "https://api.github.com/repos/pandas-dev/pandas/issues/4305",
    "id": 21302395,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxMzAyMzk1",
    "user": {
      "login": "cpcloud",
      "id": 417981,
      "node_id": "MDQ6VXNlcjQxNzk4MQ==",
      "avatar_url": "https://avatars1.githubusercontent.com/u/417981?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/cpcloud",
      "html_url": "https://github.com/cpcloud",
      "followers_url": "https://api.github.com/users/cpcloud/followers",
      "following_url": "https://api.github.com/users/cpcloud/following{/other_user}",
      "gists_url": "https://api.github.com/users/cpcloud/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/cpcloud/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/cpcloud/subscriptions",
      "organizations_url": "https://api.github.com/users/cpcloud/orgs",
      "repos_url": "https://api.github.com/users/cpcloud/repos",
      "events_url": "https://api.github.com/users/cpcloud/events{/privacy}",
      "received_events_url": "https://api.github.com/users/cpcloud/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-07-20T23:39:27Z",
    "updated_at": "2013-07-20T23:39:27Z",
    "author_association": "MEMBER",
    "body": "nope. just skipping for now if raising `IOError` (this is what the `network` decorator does), but there's something going on with a bogus url that i'm passing in tests with unicode not sure what yet\n"
  },
  {
    "url": "https://api.github.com/repos/pandas-dev/pandas/issues/comments/21302404",
    "html_url": "https://github.com/pandas-dev/pandas/pull/4305#issuecomment-21302404",
    "issue_url": "https://api.github.com/repos/pandas-dev/pandas/issues/4305",
    "id": 21302404,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxMzAyNDA0",
    "user": {
      "login": "cpcloud",
      "id": 417981,
      "node_id": "MDQ6VXNlcjQxNzk4MQ==",
      "avatar_url": "https://avatars1.githubusercontent.com/u/417981?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/cpcloud",
      "html_url": "https://github.com/cpcloud",
      "followers_url": "https://api.github.com/users/cpcloud/followers",
      "following_url": "https://api.github.com/users/cpcloud/following{/other_user}",
      "gists_url": "https://api.github.com/users/cpcloud/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/cpcloud/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/cpcloud/subscriptions",
      "organizations_url": "https://api.github.com/users/cpcloud/orgs",
      "repos_url": "https://api.github.com/users/cpcloud/repos",
      "events_url": "https://api.github.com/users/cpcloud/events{/privacy}",
      "received_events_url": "https://api.github.com/users/cpcloud/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-07-20T23:40:22Z",
    "updated_at": "2013-07-20T23:40:22Z",
    "author_association": "MEMBER",
    "body": "marking for 0.13 because i'm not sure if i'll be able to get this by this weekend\n"
  },
  {
    "url": "https://api.github.com/repos/pandas-dev/pandas/issues/comments/21525161",
    "html_url": "https://github.com/pandas-dev/pandas/pull/4305#issuecomment-21525161",
    "issue_url": "https://api.github.com/repos/pandas-dev/pandas/issues/4305",
    "id": 21525161,
    "node_id": "MDEyOklzc3VlQ29tbWVudDIxNTI1MTYx",
    "user": {
      "login": "cpcloud",
      "id": 417981,
      "node_id": "MDQ6VXNlcjQxNzk4MQ==",
      "avatar_url": "https://avatars1.githubusercontent.com/u/417981?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/cpcloud",
      "html_url": "https://github.com/cpcloud",
      "followers_url": "https://api.github.com/users/cpcloud/followers",
      "following_url": "https://api.github.com/users/cpcloud/following{/other_user}",
      "gists_url": "https://api.github.com/users/cpcloud/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/cpcloud/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/cpcloud/subscriptions",
      "organizations_url": "https://api.github.com/users/cpcloud/orgs",
      "repos_url": "https://api.github.com/users/cpcloud/repos",
      "events_url": "https://api.github.com/users/cpcloud/events{/privacy}",
      "received_events_url": "https://api.github.com/users/cpcloud/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2013-07-25T00:01:18Z",
    "updated_at": "2013-07-25T00:01:18Z",
    "author_association": "MEMBER",
    "body": "oh phooey i put in 0.12 release notes :expressionless: \n"
  }
]
